### ISSUE TITLE
vulnerability, resilience, and robustness as dispositional qualities in PATO

### ISSUE THREAD

**Issue by rlwalls2008** [pato-ontology/pato#67]
We want to be able to define classes for an application ontology such as "human population vulnerable to flooding". Such a population could be said to have a disposition to be negatively impacted by flooding, and this disposition is called vulnerability. A vulnerable disposition = vulnerability.

As PATO:quality is not really equivalent to BFO:quality and already has subclasses (like process qualities) that are not strictly BFO:qualities, we could also make a branch for dispositional qualities like vulnerable or resilient.

**Comment by pbuttigieg** [comment]
Interesting definition of resilience by the WMO [here](https://www.wmo.int/pages/prog/drr/resourceDrrDefinitions_en.html)
>Capacity to recover the normal functioning and development after being hit by a disaster. A high resilience reduces indirect impacts of disasters, such as business and services interruptions in the aftermath of a disaster. [Source: ISDR Terminology of disaster risk reduction]

Flooding and other disasters have been added to ENVO's process branch (pending release). See https://github.com/EnvironmentOntology/envo/issues/247. 
With this, PCO and the dispositions requested here, we will be much closer to expressing "human population vulnerable to flooding" and several other important classes. 
See, for example, https://github.com/SDG-InterfaceOntology/sdgio/issues/21

**Comment by cmungall** [comment]
@ruthiemusker has done a lot of work on modeling resilience and related properties (e.g. how human capacity to learn, mitigate or adapt can lead to increased resilience, or how environmental properties can also lead to resilience)

**Comment by ruthiemusker** [comment]
Hello All, 
Like Chris said, I worked with a group at ETH Zurich on how to build resilience, measure resilience, or what resilience means to various actors in the food system by breaking the concept of resilience down into "attributes". I have been playing with various methods of how organization and application of these ideas makes the most sense and I would be very happy to discuss.

**Comment by pbuttigieg** [comment]
That would be great! This will be a key entity for quite a few
socio-ecological ontologies! Would love to join this discussion / call!

**Comment by ruthiemusker** [comment]
As I am very new to ontology development, GitHub, Protege, etc (but excited to learn!), most of my information is written down in word docs, powerpoints, and spreadsheets.  Maybe it would be best to send some of this information around and then we can plan a call to discuss? Do you know of other groups are interested in these ontologies of resilience concepts? I haven't heard of many.

**Comment by pbuttigieg** [comment]
There's mounting interest from some large potential users. The
conservation, agroecology, and development crowd to name a few.

Perhaps we could set up a Google drive or dropbox folder? We can help get
your material into ontospace :)

**Comment by ruthiemusker** [comment]
I have created a google drive folder. Who shall I invite and what are their email addresses to use?

**Comment by pbuttigieg** [comment]
Please invite me with editor/invitation privileges and I'll start adding 
people.
My address is available (viewable only once) 
[here](https://privnote.com/h3sAXrL8#YU46aDYxL)

------ Original Message ------

**Comment by pbuttigieg** [comment]
@ruthiemusker Thanks! A set of invites have been sent. I don't think we need a whole ontology for resilience (as the folder name suggests), but the material is very interesting.

**Comment by rlwalls2008** [comment]
@ruthiemusker - Thanks for sharing that information! It is quite interesting and really helpful to see the work that came out of modeling agricultural sustainability.

**Comment by ruthiemusker** [comment]
You're very welcome! Please let me know if you have any questions. I am curious to see how this could build into work that is currently being done, or how it could be more useful.

**Comment by cfrancois7** [comment]
@rlwalls2008 @pbuttigieg @ruthiemusker I would like to know where are your progress in this field. I'm currently working to produce an ontology to translate the IPCC data of climate change risks into an ontological formulation (OWL). I don't want to (re)-create what is already done.
For example, did you use the last RO release in your work? Or did you use PATO?

**Comment by cmungall** [comment]
@cfrancois7 your work sounds highly relevant to the SDGIO, see: https://github.com/SDG-InterfaceOntology/sdgio

Do you have a github site for your project already?

**Comment by cfrancois7** [comment]
I already glanced SDGIO. I focused my work on the vulnerability based on BFO/RO/IAO & ENVO for the moment. And for the moment, in my ontology, the vulnerability or the risk isn't a disposition, but respectively the assessment (iao:data item) of the disposition to _be damaged_ by an hazard event and the assessment of the probability/likelihood to _be damaged_.

It is because the disposition is realized through/by a process. Be vulnerable or the risk isn't realized in a process. Also, it is not a quality because it is not fully exhibited or manifested by the material entity itself. Indeed, the /vulnerability a human concept regarding the damage potentiality of a valuable material entity.

Vulnerability is much more the measurement of a material entity to be damaged (sensitivity) in a process (classified as hazard event for this material entity) due to intrinsic quality of the material entity in certain spatiotemporal condition (exposure). This measurement could be complete with the same measurement but with adaptive capacity.

Due to my approach and the BFO standard, the SDGIO doesn't fit well for me.
